app/src/main/res/layout/activity_meal.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.design.widget.CoordinatorLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/activity_meal"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
+
+        <ImageView
+            android:id="@+id/meal_thumbnail"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/meal_page_image_height"/>
+
+        <TextView
+            android:id="@+id/meal_name"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:ellipsize="end"
+            android:maxLines="1"
+            android:layout_marginStart="@dimen/meal_page_text_margin"
+            style="@style/Base.TextAppearance.AppCompat.Headline"
+            android:layout_marginTop="@dimen/meal_page_name_margin_top"
+            android:layout_marginBottom="@dimen/meal_page_name_margin_bottom"/>
+
+        <TextView
+            android:id="@+id/meal_description"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/meal_page_text_margin"
+            android:layout_marginEnd="@dimen/meal_page_text_margin"
+            style="@style/TextAppearance.AppCompat.Body1"/>
+
+    </LinearLayout>
```

Comment:
Let's take advantage of CoordinatorLayout with some fancy-fancy animation magic ✨ . Check out this demo which involves excessive amounts of cheese: https://github.com/chrisbanes/cheesesquare/blob/master/app/src/main/res/layout/include_list_viewpager.xml.